MembershipApp/templates/inventory/DisplayCategory.html

```diff
@@ -0,0 +1,119 @@
+{% extends "MembershipApp/base.html" %}
+
+{% block header %}
+    <title>{{name}}</title>
+{% endblock %}
+
+{% block body %}
+
+    <h3 style="margin-left:1%">{{ header }}</h3>
+
+{% if category.items_type == "non_electrical_items" %}
```

Comment:
Try to always format your code, it would be easier for you or someone else to read it in the future